cross/gdb/Makefile

```diff
@@ -5,8 +5,13 @@ OPTIONAL_DEPENDS += cross/gdb-7.12
 
 include ../../mk/spksrc.main-depends.mk
 
-ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS)),$(ARCH))
+ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(PPC_ARCHS)),$(ARCH))
```

Comment:
PPC_ARCHS does build gdb-latest.  Perhaps you meant $(OLD_PPC_ARCHS) ?

Reply:
No, see my comment below for building gdb-17.2 for qoriq.

Reply:
This is now irrelevant, marking as resolved.